chat-plugins/uno.js

```diff
@@ -457,6 +457,7 @@ exports.commands = {
 			if (!room.uno.data[userid]) return false;
 			if (room.uno.pot) return this.errorReply("You cannot leave a game with bucks involved.");
```

Comment:
Shouldn't we remove this because I get the check and all but it just seems bad if we don't let users leave if they don't want to play and forces people to sit through waiting until said user is autodq'd,,,